torchsynth/module.py

```diff
@@ -455,12 +455,22 @@ class TorchFmVCO(VCO):
         modulation_index = frequency_deviation / modulation_frequency
     """
 
+    parameter_ranges: List[ModuleParameterRange] = VCO.parameter_ranges.copy()
+    parameter_ranges[1] = ModuleParameterRange(
+        0.0,
+        12.0,
+        curve=0.75,
+        name="mod_depth",
+        description="depth of the fm modulation",
+    )
+
     def make_control_as_frequency(self, midi_f0: T, mod_signal: Signal) -> Signal:
         # Compute modulation in Hz space (rather than midi-space).
         f0_hz = util.midi_to_hz(midi_f0 + self.p("tuning")).unsqueeze(1)
         fm_depth = self.p("mod_depth").unsqueeze(1) * f0_hz
         modulation_hz = fm_depth * mod_signal
-        return torch.clamp(f0_hz + modulation_hz, 0.0, self.nyquist)
+        return f0_hz + modulation_hz
```

Comment:
Turns out aliasing is feature of FM

Reply:
Make that a comment, maybe with some explanation of where you've seen this